cmd/minishift/cmd/start.go

```diff
@@ -261,6 +261,13 @@ func postClusterUp(machineName string, ip string, port int, routingSuffix string
 		atexit.Exit(1)
 	}
 
+	// Run the default addons if MINISHIFT_HOME dir does not exist or empty
+	if !homeDirExists || util.IsDirEmpty(constants.Minipath) {
```

Comment:
I think we can make the install part of root.go as well. Installation is not bound to the _start_ command. It should happen, the first time any command is executed.